conan/tools/files/patches.py

```diff
@@ -0,0 +1,107 @@
+import logging
+
+import patch_ng
+
+from conans.errors import ConanException
+
+try:
+    from collections.abc import Iterable
+except ImportError:
+    from collections import Iterable
+
+
+class PatchLogHandler(logging.Handler):
+    def __init__(self, conanfile, patch_file):
+        logging.Handler.__init__(self, logging.DEBUG)
+        self._output = conanfile.output
+        self.patchname = patch_file if patch_file else "patch_ng"
```

Comment:
why is it needed? also, could be simplified
```suggestion
        self.patchname = patch_file or "patch_ng"
```

Reply:
True. I think I just copy-pasted from the _legacy_ implementation without looking at the actual statement.